x-pack/platform/plugins/shared/cases/server/client/cases/bulk_update.test.ts

```diff
@@ -315,6 +315,90 @@ describe('update', () => {
         'Failed to update case, ids: [{"id":"mock-id-1","version":"WzAsMV0="}]: Error: The length of the field assignees is too long. Array must be of length <= 10.'
       );
     });
+
+    it('returns only updateCase operation when no reopened cases or changed assignees', () => {
```

Comment:
What do you think instead of testing only the util function to test the whole flow with cases that do different actions and check how the ensured authorized function was called?